app/views/invite_codes/_invite_codes.html.erb

```diff
@@ -0,0 +1,22 @@
+<%# app/views/invite_codes/_invite_codes.html.erb %>
+<div id="invite-codes">
+  <div class="space-y-2">
+    <% @invite_codes.each do |invite_code| %>
+      <div class="flex items-center" data-controller="clipboard">
+        <span class="flex-shrink-0 z-10 inline-flex items-center py-2.5 px-3 text-sm font-medium text-center text-gray-900 bg-gray-200 border border-2 border-dashed border-gray-500 border-r-0 rounded-s-lg">CODE</span>
+        <div>
+          <input type="text" autocomplete="off" data-clipboard-target="source" class="border border-2 border-dashed border-gray-500 border-l-0 border-r-0 bg-gray-100 text-gray-900 text-base text-center font-bold p-2 w-32" value="<%= invite_code %>" readonly disabled>
+        </div>
+        <button data-action="clipboard#copy" class="flex-shrink-0 z-10 inline-flex items-center py-2.5 px-3 text-sm font-medium text-center text-white bg-gray-900 border border-2 border-dashed border-gray-800 border-l-0 rounded-e-lg hover:bg-gray-900 focus:outline-none" type="button">
+          <span data-clipboard-target="iconDefault">
+            <%= lucide_icon "clipboard-list", class: "w-5 h-5" %>
+          </span>
+          <span class="hidden inline-flex items-center" data-clipboard-target="iconSuccess">
+            <%= lucide_icon "check", class: "w-5 h-4" %>
+            <span class="text-sm w-fit">copied</span>
+          </span>
+        </button>
+      </div>
+    <% end %>
+  </div>
+</div>
```

Comment:
Code looks good here, but can we move this to a `_invite_code.html.erb` partial, and then in `index` we can render a collection?

```
<%= render @invite_codes %>
```